funcx/sdk/client.py

```diff
@@ -1,24 +1,27 @@
-from funcx.sdk.config import (check_logged_in, FUNCX_SERVICE_ADDRESS, CLIENT_ID, lookup_option)
-from funcx.sdk.utils.auth import do_login_flow, make_authorizer, logout
-from funcx.sdk.utils.futures import FuncXFuture
```

Comment:
Removing this line `from funcx.sdk.utils.futures import FuncXFuture` gives the error `module FuncXFuture not found`

Reply:
Should be fixed now, can you take a quick look ?

Reply:
Now it gives a different error on this line
```
--> 134         funcx_future = FuncXFuture(self, task_id, async_poll)
    135 
    136         if not asynchronous:

NameError: name 'async_poll' is not defined
```

Reply:
Can you reinstall and check, please ?

Reply:
This is now fixed.